lib/puppet/provider/reboot/linux.rb

```diff
@@ -1,3 +1,5 @@
+require_relative '../reboot/posix'
```

Comment:
Is this module supported on older puppet versions that bundle ruby 1.8.7? If so, then we can't use require_relative.

Since linux and posix are in the same directory, you could require `relative_relative './posix'`.